bundles/ch.docbox.elexis/src/ch/docbox/elexis/DocboxView.java

```diff
@@ -103,10 +103,9 @@ public void setAppointment(String termin) {
 	private String getSSOLoginParams(String page) {
 		String ts = StringUtils.EMPTY + System.currentTimeMillis() / 1000;
 		String username = UserDocboxPreferences.getDocboxLoginID(false);
-		String signature = UserDocboxPreferences.getSSOSignature(ts);
 		try {
-			return "?ts=" + ts + "&loginId=" + URLEncoder.encode(username, "UTF-8") + "&sig="
-					+ URLEncoder.encode(signature, "UTF-8") + "&page=" + URLEncoder.encode(page, "UTF-8");
+			return "?ts=" + ts + "&loginId=" + URLEncoder.encode(username, "UTF-8") + "&page="
+					+ URLEncoder.encode(page, "UTF-8");
```

Comment:
Wieso hat sich der code hier geändert. Gibt es da ein Hinweis in der Doku das es die signature nicht mehr braucht?